dispatch.yaml

```diff
@@ -4,6 +4,9 @@ dispatch:
 # Route the urls that point to the Java backend's API calls
  - url: "*/api/v1/*"
    service: backend
+# Route the urls that point to the Java backend's API calls
```

Comment:
Yes, the `/blackswan/` url should also point to Java backend's API calls since they're in the same maven project.
However, i think the comments are a bit redundant, so i will fix that. thanks!